Sources/CvGameInterface.cpp

```diff
@@ -660,14 +660,12 @@ void CvGame::updateColoredPlots()
 						{
 							if (pHeadSelectedUnit->canFound(pLoopPlot))
 							{
-// BUFFY - Don't Recommend Plots in Fog of War - start
-#ifdef _BUFFY
-								if (!pLoopPlot->isVisible(pHeadSelectedUnit->getTeam(), false))
-								{
-									continue;
-								}
-#endif
-// BUFFY - Don't Recommend Plots in Fog of War - end
+								// Don't Recommend Plots in Fog of War - start
```

Comment:
Is this the correct fix here? I don't know the context but is the _BUFFY change here an improvement, and if so why not keep it enabled?